solidity/contest/FreeTonContest.sol

```diff
@@ -181,51 +163,56 @@ contract FreeTonContest is IContestData {
     }
 
     /* Combined juror/entry id to single out a mark */
-    function _mid(uint8 jid, uint16 eid) private inline pure returns (uint24) {
-        return uint24(jid * (1 << 16) + eid);
+    function _mid(uint8 jid, uint8 eid) private inline pure returns (uint16) {
+        return uint16(jid * (1 << 8) + eid);
```

Comment:
Is it ok that you reduce range of `eid` from 16 bytes to 8 ?  It can be only 256 entries.

Reply:
Yes. That's a basic version to build upon. More than 255 entries would likely require special handling.